lib/components/UploadPicker.vue

```diff
@@ -86,13 +91,13 @@ import Cancel from 'vue-material-design-icons/Cancel.vue'
 import Plus from 'vue-material-design-icons/Plus.vue'
 import Upload from 'vue-material-design-icons/Upload.vue'
 
-import { getUploader } from '../index.ts'
+import { getUploader, openConflictPicker } from '../index.ts'
 import { Status } from '../uploader.ts'
 import { Status as UploadStatus } from '../upload.ts'
 import { t } from '../utils/l10n.ts'
 import logger from '../utils/logger.ts'
 
-export default {
+export default Vue.extend({
```

Comment:
Why is it needed ? Typing ?

Reply:
Yes, typescript :)
We do the same in server